docs/configuration/alertmanager-integration/grafana-alert-manager.rst

```diff
@@ -1,31 +1,108 @@
 Grafana AlertManager
 ****************************************
 
-If you use Grafana alerts, you're using a `special instance of AlertManager embedded inside Grafana <https://grafana.com/docs/grafana/latest/alerting/fundamentals/alertmanager/>`_.
+Using Grafana alerts involves a `special instance of AlertManager embedded within Grafana <https://grafana.com/docs/grafana/latest/alerting/fundamentals/alertmanager/>`_.
 
-You will need to configure to send Grafana alerts to Robusta.
+You will need to configure two integrations one to send alerts to Robusta and another to let Robusta query metrics and create silences.
 
 Send Alerts to Robusta
 ============================
 
-.. include:: ./_alertmanager-config.rst
+This integration lets you send Grafana alerts to Robusta.
 
-.. include:: ./_pull_integration.rst
+To configure it:
 
-Configure Alert Silencing
-==============================
-For Robusta to create silences, you need to modify one additional setting. This is necessary because of minor API differences in the AlertManager embedded in Grafana.
+1. In your Grafana cloud site, go to Alerting, then click ``Add contact point`` and in the Integration section, choose ``Webhook``.
+2. Insert the following URL:
 
-Add the following configuration to the ``globalConfig`` section in your ``generated_values.yaml`` file:
+.. code-block::
 
-.. admonition:: generated_values.yaml
+    https://api.robusta.dev/integrations/generic/alertmanager
 
-    .. code-block:: yaml
+3. Change the HTTP Method to POST in the ``Optional Webhook Settings``.
+4. Enter your "Account_id Signing_key" in the ``Authorization Headers - Scheme`` box (locate this in your generated_values.yaml file).
+5. Click "Test" and you should receive a notification on Robusta UI.
+6. Finally, click "Save contact point" to complete the Robusta integration.
 
-        globalConfig: # this line should already exist
-          grafana_api_key: <YOUR GRAFANA EDITOR API KEY>
-          alertmanager_flavor: grafana
+.. note::
 
-    .. note::
+  Ensure each cluster has a ``cluster_name`` label.
```

Comment:
This is really important and non-trivial. Can you please make it more obvious. E.g. list it under a prerequisites section include something in the Test instructions for this. (If we were unable to figure out how to test at first, what chance does the user have? Help them by telling them what we now know.)